qiskit/visualization/circuit_visualization.py

```diff
@@ -444,7 +444,10 @@ def _latex_circuit_drawer(
                 image = utils._trim(image)
                 os.remove(base + ".png")
                 if filename:
-                    image.save(filename, "PNG")
+                    if filename.endswith(".pdf"):
+                        os.rename(base + ".pdf", filename)
```

Comment:
Maybe I'm missing something obvious, but shouldn't this save something? 🤔

Reply:
it's moving the temporary PDF file (the result of the latex compilation) to the user location. The file exists. It's just out of place :P